storage/src/main/java/tech/pegasys/teku/storage/server/leveldb/LevelDbInstance.java

```diff
@@ -91,9 +92,18 @@ public LevelDbInstance(
 
   @Override
   public <T> Optional<T> get(final KvStoreVariable<T> variable) {
+    return getRaw(variable)
+        .map(value -> variable.getSerializer().deserialize(value.toArrayUnsafe()));
+  }
+
+  @Override
+  public Optional<Bytes> getRaw(final KvStoreVariable<?> variable) {
     assertOpen();
-    return Optional.ofNullable(db.get(getVariableKey(variable)))
-        .map(variable.getSerializer()::deserialize);
+    final byte[] bytes = db.get(getVariableKey(variable));
+    if (bytes == null) {
+      return Optional.empty();
+    }
+    return Optional.of(Bytes.wrap(bytes));
```

Comment:
Either way is fine but personally I'd write this as:
```suggestion
    return Optioanl.ofNullable(db.get(getVariableKey(variable)).map(Bytes::wrap);
```

Reply:
hrm i had that change, i thought i pushed it... :/